app/scripts/controllers/util/error.js

```diff
@@ -40,4 +42,29 @@ angular.module('openshiftConsole')
     $scope.reloadConsole = function() {
       $window.location.href = "/";
     };
+
+    function checkHomePagePreference(projectName) {
```

Comment:
Curious if this function actually belongs within `HomePagePreference` rather than in `ErrorController`, even if it gets called here.  Should this module grow into a collection of error functions related to other modules?